src/env.ts

```diff
@@ -2,13 +2,19 @@
 import { config } from './config';
 import type { Canvas } from 'canvas';
 
+type TCopyPasteData = {
+  copiedText?: string;
+  copiedStyle?: Record<string, string>;
+};
+
 type TFabricEnv = {
   document: Document;
   window: Window;
   isTouchSupported: boolean;
   isLikelyNode: boolean;
   nodeCanvas: Canvas;
   jsdomImplForWrapper: any;
+  copyPasteData: TCopyPasteData;
```

Comment:
this should die in a PR I need to port

Reply:
if different copy paste code make this unnecessary, great.

Reply:
yes, using the native event `transferData` is the path forward